proto/steward/v4/cellar_v2.proto

```diff
@@ -231,6 +248,19 @@ message CellarV2_2 {
             SetStrategistPayoutAddress set_strategist_payout_address = 7;
             // Represents function `swapPositions(uint256 index1, uint256 index2)`
             SwapPositions swap_positions = 8;
+
+            // TEMPORARY
+            // These are governance functions, but will be allowed in a limited capacity for SPs while
+            // the new governance model is still in early adoption for emergencies.
+
+            // Represents function `cachePriceRouter(bool checkTotalAssets, uint16 allowableRange, address expectedPriceRouter)`
+            CachePriceRouter cache_price_router = 9;
+            // Represents function `addAdaptorToCatalogue(address adaptor)`
+            AddAdaptorToCatalogue add_adaptor_to_catalogue = 10;
+            // Represents function `addPositionToCatalogue(uint32 positionId)`
+            AddPositionToCatalogue add_position_to_catalogue = 11;
+            // Represents function `forcePositionOut(uint32 index, uint32 positionId, bool inDebtArray)`
+            ForcePositionOut force_position_out = 12;
```

Comment:
This should be a gov function I believe. We don't have any exception rules to deal with it.

Reply:
OK actually we do, but look at the later comment for discussion.

Reply:
3.x has one valid call for Turbo SOMM; it seemed similar to other exceptions we are porting:

https://github.com/PeggyJV/steward/blob/6ccf3bd7b1affaa1eca2ac7c4875529e1d744236/steward/src/cellars.rs#L304-L322